ecs-cli/modules/utils/waiters/waiters.go

```diff
@@ -34,23 +34,22 @@ const (
 	// servicesWaitDelay is the delay between successive ECS DescribeServices API calls
 	// while determining if the service is stable or inactive. This value
 	// reflects the values set in the ecs waiters json file in the aws-go-sdk.
-	servicesWaitDelay = 15 * time.Second
-
-	// servicesMaxRetries is the maximum number of ECS DescribeServices API will be invoked by the WaitUntilComplete method
-	// to determine if the task is running or stopped before giving up. This value reflects the values set in the
-	// ecs waiters json file in the aws-go-sdk.
-	servicesMaxRetries = 40
+	servicesWaitDelay = 5 * time.Second
```

Comment:
seems like this is not used at all in the entire codebase?

Reply:
It is here: https://github.com/aws/amazon-ecs-cli/pull/996/files#diff-1c2be9e693da48c06ad67f1a3fd29bbdL60

Reply:
lol got it. Github search failed on me again